packages/forma-36-react-components/src/components/Switch/Switch.css

```
@@ -35,17 +35,17 @@
 
 .Switch--small {
   width: calc(1rem * (28 / var(--font-base-default)));
-  height: calc(1rem * (12 / var(--font-base-default)));
+  height: calc(1rem * (14 / var(--font-base-default)));
 
   & + .Switch__label {
     & .Switch__icon--done {
-      top: 3px;
-      left: 3px;
+      top: 0.15rem;
```

the top isn't needed, if we remove it from the icons it should be centered horizontally

```suggestion
      top: 0.15rem;
```